yt_dlp/extractor/nekohacker.py

```diff
@@ -0,0 +1,199 @@
+                'md5': '8f853c71719389d32bbbd3f1a87b3f08',
+                'info_dict': {
+                    'id': '1713',
+                    'ext': 'mp3',
+                    'title': 'City Runner',
+                    'thumbnail': 'https://nekohacker.com/wp-content/uploads/2022/11/Nekoverse_Artwork-1024x1024.jpg',
+                    'vcodec': 'none',
+                    'acodec': 'mp3',
+                    'release_date': '20221101',
+                    'album': 'Nekoverse',
+                    'artist': 'Nekohacker',
+                    'track': 'City Runner',
+                    'track_number': 2,
+                    'duration': 148.0
+                }
+            },
+            {
+                'url': 'https://nekohacker.com/wp-content/uploads/2022/11/03-Nature-Talk.mp3',
+                'md5': '5a8a8ae852720cee4c0ac95c7d1a7450',
+                'info_dict': {
+                    'id': '1714',
+                    'ext': 'mp3',
+                    'title': 'Nature Talk',
+                    'thumbnail': 'https://nekohacker.com/wp-content/uploads/2022/11/Nekoverse_Artwork-1024x1024.jpg',
+                    'vcodec': 'none',
+                    'acodec': 'mp3',
+                    'release_date': '20221101',
+                    'album': 'Nekoverse',
+                    'artist': 'Nekohacker',
+                    'track': 'Nature Talk',
+                    'track_number': 3,
+                    'duration': 174.0
+                }
+            },
+            {
+                'url': 'https://nekohacker.com/wp-content/uploads/2022/11/04-Crystal-World.mp3',
+                'md5': 'd8e59a48061764e50d92386a294abd50',
+                'info_dict': {
+                    'id': '1715',
+                    'ext': 'mp3',
+                    'title': 'Crystal World',
+                    'thumbnail': 'https://nekohacker.com/wp-content/uploads/2022/11/Nekoverse_Artwork-1024x1024.jpg',
+                    'vcodec': 'none',
+                    'acodec': 'mp3',
+                    'release_date': '20221101',
+                    'album': 'Nekoverse',
+                    'artist': 'Nekohacker',
+                    'track': 'Crystal World',
+                    'track_number': 4,
+                    'duration': 199.0
+                }
+            }
+        ]
+    }, {
+        'url': 'https://nekohacker.com/susume/',
+        'info_dict': {
+            'id': 'susume',
+            'title': '進め！むじなカンパニー',
+        },
+        'playlist': [
+            {
+                'url': 'https://nekohacker.com/wp-content/uploads/2021/01/進め！むじなカンパニー-feat.-六科なじむ-CV_-日高里菜-割戶真友-CV_-金元寿子-軽井沢ユキ-CV_-上坂すみれ-出稼ぎガルシア-CV_-金子彩花-.mp3',
+                'md5': 'fb13f008aa81f26ba48f91fd2d6186ce',
+                'info_dict': {
+                    'id': '711',
+                    'ext': 'mp3',
+                    'title': '進め！むじなカンパニー feat. 六科なじむ (CV: 日高里菜 ) & 割戶真友 (CV: 金元寿子 ) & 軽井沢ユキ (CV: 上坂すみれ ) & 出稼ぎガルシア (CV: 金子彩花 )',
```

Comment:
use md5 for long titles/descriptions/track titles

Reply:
do you have a example of how to do this (another extractor) or which function to use?

Reply:
change the value to `'md5:TODO'`, run the test again, it will fail and output the hash value

Reply:
done with 5d8ef9cbc282f616b0638b12f80d4039fae29998